libraries/botbuilder-core/tests/turnContext.test.js

```diff
@@ -317,31 +291,24 @@ describe(`TurnContext`, function () {
             called = true;
             return next();
         });
-        context.deleteActivity({ activityId: '1234' }).then((responses) => {
-            assert(called, `delete hook not called.`);        
-            done();
-        });
+        await context.deleteActivity({ activityId: '1234' });
+        assert(called, `delete hook not called.`);
     });
-    
-    it(`should map an exception raised by a hook to a rejection.`, function (done) {
+
+    it(`should map an exception raised by a hook to a rejection.`, async function () {
         let called = false;
         const context = new TurnContext(new SimpleAdapter(), testMessage);
         context.onDeleteActivity((ctx, reference, next) => {
             throw new Error('failed');
         });
-        context.deleteActivity('1234')
-            .then((responses) => {
-                assert(false, `exception swallowed.`);        
-            })
-            .catch((err) => {
-                assert(err, `invalid exception returned.`);        
-                done();
-            });
+        await assert.rejects(async () => await context.deleteActivity('1234'), {
+            message: 'failed',
+        });
     });
```

Comment:
I could use a second set of eyes on this one. Not exactly sure what it's trying to test and if my refactor still tests for the same thing.

Reply:
This looks correct to me. Still passing?

Reply:
Yes, it still passes. Just wanted to be sure I refactored it appropriately.